sources/TileDB.CSharp/CoreUtil.cs

```diff
@@ -3,9 +3,23 @@
 using System;
 using System.Text;
 using System.Collections.Generic;
+using System.IO;
+
 namespace TileDB.CSharp {
 
-    public class CoreUtil {
+    public static class CoreUtil {
```

Comment:
You should add this kind of code to a utilities file outside the main `TileDB.CSharp` library.

And making a class `static` is a breaking change because it does not allow constructing it and inheriting it (not that anybody would do these here but anyway).

Reply:
Done :heavy_check_mark: